### PR TITLE
Updated connectionTtl to be 120 seconds

### LINKED ISSUE #1243: Update connectionStateTtl
- Update connectionStateTtl to 2 mins as per spec.

### PR DESCRIPTION
**Pull request by sacOO7**
- Fixed #1243

### CODE REVIEW

File: src/IO.Ably.Tests.Shared/Realtime/ConnectionSandBoxSpecs.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ public async Task ShouldSaveConnectionStateTtlToConnectionObject(Protocol protoc
 
             await WaitForState(client);
 
-            client.Connection.ConnectionStateTtl.Should().NotBe(Defaults.ConnectionStateTtl);
+            client.Connection.ConnectionStateTtl.Should().Be(Defaults.ConnectionStateTtl);
```
Comment (owenpearson) [inline src/IO.Ably.Tests.Shared/Realtime/ConnectionSandBoxSpecs.cs:92]: This test change isn't quite right - what this test does is ensure that once a client is connected, the `Connection.ConnectionStateTtl` field is populated with the `connectionStateTtl` from connection details. I'm not sure how easy this is to do with dotnet tests, but to test this properly you need to change the value of `Defaults.ConnectionStateTtl` and assert that the value of `Connection.ConnectionStateTtl` changes when you become connected (as the test was doing before).

**Review by owenpearson — CHANGES_REQUESTED** [review]

File: src/IO.Ably.Tests.Shared/Realtime/ConnectionSandBoxSpecs.cs (hunk 1 above)
Comment (sacOO7) [reply src/IO.Ably.Tests.Shared/Realtime/ConnectionSandBoxSpecs.cs:92]: Then this test should be removed wdyt? I think currently we are just checking if test doesn't fail.

Comment (sacOO7) [reply src/IO.Ably.Tests.Shared/Realtime/ConnectionSandBoxSpecs.cs:92]: Also. we haven't marked the test with any spec id yet.

Comment (owenpearson) [reply src/IO.Ably.Tests.Shared/Realtime/ConnectionSandBoxSpecs.cs:92]: I don't think we should remove the test, the behaviour is defined by [`DF1a`](https://sdk.ably.com/builds/ably/specification/main/features/#DF1a).

If we don't test this and someone breaks it then we could end up in a situation where we change the server-side `connectionStateTtl` but ably-dotnet clients just ignore the new value, therefore it's important to test that the `connectionStateTtl` from connection details is used

Comment (sacOO7) [reply src/IO.Ably.Tests.Shared/Realtime/ConnectionSandBoxSpecs.cs]: Updated test as per suggestion.

**Pull request merged**